trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -13,6 +13,8 @@
 public class GravitinoConfig {
 
   public static String GRAVITINO_DYNAMIC_CONNECTOR = "__gravitino.dynamic.connector";
+  public static String GRAVITINO_DYNAMIC_CONNECTOR_CATALOG_CONFIG =
+      "__gravitino.dynamic.connector.catalog.config";
```

Comment:
Is the value of this key set automatically determined by Trino?

Reply:
Yes